Serving with turbo fails when node-cron is used: Server relative imports are not implemented yet

### Link to the code that reproduces this issue

https://github.com/klein-aber/minimal-example-node-corn-turbo

### To Reproduce

We have node cron in our next project, that we setup in instrumentation.ts. It is just one, very simple task.
However we need to switch to tubo asap, because otherwise our compilation times in dev are painfully slow (even on well equipped hardware). The thing that stops us from doing this is an error related to the node-cron module when running yarn dev --turbo:

```
 ⨯ ./node_modules/node-cron/src/background-scheduled-task
Module not found: Can't resolve '/ROOT/node_modules/node-cron/src/background-scheduled-task/daemon.js'
```

I have seen some related issues, but that was related to importing static assets. Is there any way I can fix this on my end except removing node-cron?

I originally posted this in turbo, because I have seen that issues related to turbo were moved to the turbo repo in the past, but turbo moved it here.

### Current vs. Expected behavior

Expected: Serve the app with turbo
Current behaviour: App crashes 

### Provide environment information

```bash
Tested it on many different systems (Mac, Docker with Ubuntu, Ubuntu on machine..) , its the same behaviour everywhere.
```


### Which area(s) are affected? (Select all that apply)

Turbopack

### Which stage(s) are affected? (Select all that apply)

next dev (local)

### Additional context

_No response_

The full error message with the latest canary is
```
 ⨯ ./node_modules/.pnpm/node-cron@3.0.3/node_modules/node-cron/src/background-scheduled-task
Module not found: Can't resolve '/ROOT/node_modules/.pnpm/node-cron@3.0.3/node_modules/node-cron/src/background-scheduled-task/daemon.js'
server relative imports are not implemented yet. Please try an import relative to the file you are importing from.


https://nextjs.org/docs/messages/module-not-found


 ⨯ ./node_modules/.pnpm/node-cron@3.0.3/node_modules/node-cron/src/background-scheduled-task/index.js:29:28
Module not found: Can't resolve '/ROOT/node_modules/.pnpm/node-cron@3.0.3/node_modules/node-cron/src/background-scheduled-task/daemon.js'
  27 |     start() {
  28 |         this.stop();
> 29 |         this.forkProcess = fork(daemonPath);
     |                            ^^^^^^^^^^^^^^^^
  30 |
  31 |         this.forkProcess.on('message', (message) => {
  32 |             switch(message.type){



https://nextjs.org/docs/messages/module-not-found
```

--- a/docs/02-app/02-api-reference/05-next-config-js/serverExternalPackages.mdx
+++ b/docs/02-app/02-api-reference/05-next-config-js/serverExternalPackages.mdx
@@ -54,6 +54,7 @@ Next.js includes a [short list of popular packages](https://github.com/vercel/ne
 - `mongoose`
 - `next-mdx-remote`
 - `next-seo`
+- `node-cron`
 - `node-pty`
 - `node-web-audio-api`
 - `oslo`

--- a/docs/03-pages/02-api-reference/04-next-config-js/serverExternalPackages.mdx
+++ b/docs/03-pages/02-api-reference/04-next-config-js/serverExternalPackages.mdx
@@ -54,6 +54,7 @@ Next.js includes a [short list of popular packages](https://github.com/vercel/ne
 - `mongoose`
 - `next-mdx-remote`
 - `next-seo`
+- `node-cron`
 - `node-pty`
 - `node-web-audio-api`
 - `oslo`

--- a/packages/next/src/lib/server-external-packages.json
+++ b/packages/next/src/lib/server-external-packages.json
@@ -35,6 +35,7 @@
   "mongoose",
   "next-mdx-remote",
   "next-seo",
+  "node-cron",
   "node-pty",
   "node-web-audio-api",
   "oslo",

